### PR TITLE
lint ALL-THE-THINGS!

### PR DESCRIPTION
**Pull request by btardif**
I did a full review of the repository, there are many things that need to be cleaned up including:

- **Spelling Mistakes:** Spelling mistakes are embarrassing and erode the confidence in the docs.
- **Linting/ Styling / Formatting:** Violating markdown styling rules makes it harder to maintain and have consistency across authors/editors
- **Screenshots:** Same as Spelling mistakes, having high quality screenshots helps create a good impression.

There are a few tools that I recommend (my main editor for markdown is VSCODE)

- [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
- [Markdown Lint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)
- [Paste Image](https://marketplace.visualstudio.com/items?itemName=mushan.vscode-paste-image)
- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)

For screenshots I use [Screenpresso](https://www.screenpresso.com) but you can use [Flameshot](https://flameshot.org/) as a free alternative. It includes image annotation tools to avoid freehand drawing.

I'll drop some comments inline below for a few TODO's that I identified in the docs, they should not block this PR but it's stuff we should look at soon.

CC: @ZubaeyrMSFT / @rabollin / @saisubodhMSFT

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -10,16 +10,13 @@
 
 # Support
 
-## How to file issues and get help
```
Comment (btardif) [inline SUPPORT.md:13]: This whole file is just a placeholder, @rabollin you guys checking in a template. Please read through and fill it out as needed

File: WordPress/changing_mysql_database_password.md [hunk 2]
```diff
@@ -13,14 +13,12 @@ Please note that the Application Settings related to database connection are **p
...
 
-### Changing MySQL Database Password
-
-First, go to the MySQL resource corresponding to your WordPress deployment, and click on **Reset Password** option as shown below. Now enter the new password and click on Save. Wait until the action is completed. 
+Then navigate to the Configuration section of your AppService and update the **DATABASE_PASSWORD** Application Settings in your AppService. Once you update the value, click on Save and wait for app to get restarted.
 
-Then navigate to the Configuration section of your AppService and update the **DATABASE_PASSWORD** Application Settings in your AppService. Once you update the value, click on Save and wait for app to get restarted. 
+![Password Rest](./media/changing_mysql_password_1.png)
 
-<br>
```
Comment (btardif) [inline WordPress/changing_mysql_database_password.md:24]: @rabollin we should not be in-lining html, this was a common mistake throughout the repo

File: WordPress/running_post_startup_scripts.md [hunk 3]
```diff
@@ -24,22 +24,26 @@ A custom startup script has many use cases. The following are some scenarios for
 
 Configurations for nginx are defined in **/etc/nginx/conf.d/default.conf** and **/etc/nginx/conf.d/spec-settings.conf** files. You can update any of these files using a startup script. For instance, the following commands in startup script changes the nginx configuration for max simultaneous connections to 20000.
 
-``` 
+``` bash
```
Comment (btardif) [inline WordPress/running_post_startup_scripts.md:27]: @rabollin all code blocks should specify the target language, this help render rich formatting

File: WordPress/wordpress_application_settings.md [hunk 4]
```diff
@@ -1,59 +1,50 @@
...
+|DATABASE_NAME| Database| -| -| Database name used to connect to WordPress.|
+|DATABASE_USERNAME| Database| -| -| Database username used to connect to WordPress.|
+|DATABASE_PASSWORD| Database| -| -| Database password used to connect to WordPress.|
 
 ## Non-Persistent Application Settings
 
-Note that these are used only once during the installation process and any subsequent changes to these application settings after installation will not impact the WordPress application.
+**Non-Persistent** Application Settings which are used as a 'one-time' reference during the installation process. Any subsequent changes to these App Settings after the installation is complete will not update the same in WordPress application.
 
+<!TODO: We probably need a better name than *non-persistent* we should also clarify if these can be deleted, or should be deleted as a best practice after initial setup.>
```
Comment (btardif) [inline WordPress/wordpress_application_settings.md:29]: @rabollin see the comment in the TODO

File: WordPress/wordpress_migration_linux_appservices.md [hunk 5]
```diff
@@ -1,134 +1,125 @@
...
-2. Open Bash Shell from the menu shown on the top.
-3. Delete the existing content of **/home/site/wwwroot/wp-content** folder using the following command.
-    ````
-    rm -rf /home/site/wwwroot/wp-content/* 
-    ````
-4. Now upload the new contents of **wp-content** folder using the File Manager. Click on the label that says '**Drag a File/Folder here to upload, or click to select one**'. Please note that if you are not able to upload everything at once, then you can try dividing your upload into multiple smaller ones.
+1. Upload the new contents of **wp-content** folder using the File Manager. Click on the label that says '**Drag a File/Folder here to upload, or click to select one**'. Please note that if you are not able to upload everything at once, then you can try dividing your upload into multiple smaller ones.
 
-5. You can either point your WordPress to [use an existing MySQL database](./using_an_existing_mysql_database.md), or use the steps below to migrate the content to the new database server created by the WordPress on Linux App Services offering.
+   <!TODO: we should prob recomend doing this over FTP, it's more reliable than web upload through SCM site?>
```
Comment (btardif) [inline WordPress/wordpress_migration_linux_appservices.md:71]: @rabollin see my comment in the TODO

File: WordPress/changing_mysql_database_password.md (hunk 2 above)
Comment (rabollin) [reply WordPress/changing_mysql_database_password.md:24]: Agreed

File: WordPress/running_post_startup_scripts.md (hunk 3 above)
Comment (rabollin) [reply WordPress/running_post_startup_scripts.md:27]: taken care of.

**Review by rabollin — APPROVED** [review]

**Review by rabollin — APPROVED** [review]

File: WordPress/wordpress_application_settings.md (hunk 4 above)
Comment (rabollin) [reply WordPress/wordpress_application_settings.md:29]: Fixed it

File: WordPress/wordpress_migration_linux_appservices.md (hunk 5 above)
Comment (rabollin) [reply WordPress/wordpress_migration_linux_appservices.md:71]: @btardif Will review and raise a separate PR for this

File: SUPPORT.md (hunk 1 above)
Comment (rabollin) [reply SUPPORT.md:13]: sure

**Pull request merged**